packages/api/tests/endpoints/test-elasticsearch.js

```diff
@@ -222,6 +222,24 @@ test.serial('Reindex - specify a source index that is not aliased', async (t) =>
   await esClient.indices.delete({ index: destIndex });
 });
 
+test.serial('Reindex fails when source index matches destination index.', async (t) => {
```

Comment:
🐜 - slight preference for expanding on 'fail' when not something like `t.throws`

```suggestion
test.serial('Reindex request returns 400 with the expected message when source index matches destination index.', async (t) => {
```